lib/runners/csharp.js

```diff
@@ -89,41 +89,6 @@ module.exports.run = function run(opts, cb) {
       // compile
       compile(args, function(error, stdout, stderr) {
         if (error) return fail(error, stdout, stderr);
-
-        const exe = path.join(opts.dir, 'runner.exe');
-
-        // compile([
```

Comment:
Ok, I'll just comment out that `const exe = path.join(opts.dir, 'runner.exe');`.